docs/core/build/distribution-packaging.md

```diff
@@ -72,48 +91,28 @@ The rest of the version isn't included in the version name. This allows the OS p
-| dotnet-sdk-[major].[minor].[sdk feat]xx | dotnet-sdk-2.1.3xx     | Specific sdk feature release    | (3),(4)            | aspnetcore-runtime-[major].[minor]             | \<sdk version>     |
-| aspnetcore-runtime-[major].[minor]      | aspnetcore-runtime-2.1 | Specific ASP.NET Core runtime   | (6),[(7)]          | dotnet-runtime-[major].[minor]                 | \<runtime version> |
-| dotnet-runtime-[major].[minor]          | dotnet-runtime-2.1     | Specific runtime                | (5)                | host-fxr:\<runtime version>+                   | \<runtime version> |
-| dotnet-host-fxr                         | dotnet-host-fxr        | _dependency_                    | (2)                | host:\<runtime version>+                       | \<runtime version> |
-| dotnet-host                             | dotnet-host            | _dependency_                    | (1),(8),(9),(10)   |                                                | \<runtime version> |
+| Name                              | Example            | Use case: Install ...           | Contains           | Dependencies                                   | Version            |
+|-----------------------------------|--------------------|---------------------------------|--------------------|------------------------------------------------|--------------------|
+| dotnet-sdk-[major]                | dotnet-sdk-2       | Latest sdk for runtime major    |                    | dotnet-sdk-[major].[latestminor]               | \<sdk version>     |
+| dotnet-sdk-[major].[minor]        | dotnet-sdk-2.1     | Latest sdk for specific runtime | (3),(4),(11),(12)  | aspnet-runtime-[major].[minor], dotnet-netstd-pack-[netstd_major].[netstd_minor] | \<sdk version>     |
+| aspnet-runtime-[major].[minor]    | aspnet-runtime-2.1 | Specific ASP.NET Core runtime   | (6),[(7)]          | dotnet-runtime-[major].[minor]                 | \<runtime version> |
+| dotnet-runtime-[major].[minor]    | dotnet-runtime-2.1 | Specific runtime                | (5)                | host-fxr:\<runtime version>+                   | \<runtime version> |
+| dotnet-host-fxr                   | dotnet-host-fxr    | _dependency_                    | (2)                | host:\<runtime version>+                       | \<runtime version> |
+| dotnet-host                       | dotnet-host        | _dependency_                    | (1),(8),(9),(10)   |                                                | \<runtime version> |
+| dotnet-desktop-pack-[major.minor] |                    | _dependency_                    | (12)               |                                                | \<desktop version>  |
+| dotnet-netstd-pack-[major.minor]  |                    | _dependency_                    | (13)               |                                                | \<netstd version>  |
```

Comment:
These are called `netstandard-targeting-pack` and `windowsdesktop-targeting-pack` in Microsoft builds. I can see how a prefix would be a good idea for discoverability (especially when on a "real" package repository) but I'm not a fan of the abbreviations here.

`netstd`: I haven't seen this before, I searched and found that we don't want to use it: https://github.com/dotnet/corefx/pull/4362#discussion-diff-44104229.

`pack`: there are multiple kinds of packs, only including `pack` in the package name could be confusing. In particular, someone installing this might expect runtime packs.

`desktop`: well, we don't currently plan to make this targeting pack available on Linux because there are some build-time Windows dependencies: https://github.com/dotnet/core-setup/pull/6198. If it somehow became a critical scenario to allow someone to target WindowsDesktop when building on Linux, though, including `windows` seems like a good idea. It would match the name of the framework, and avoids being misinterpreted into a false expectation that it will work on non-Windows platforms.

Reply:
Thanks @dagood. I've added separate packages for aspnet and netcoreapp, and changed the naming a bit.

> desktop: well, we don't currently plan to make this targeting pack available on Linux because there are some build-time Windows dependencies:

We need to look into this. The 3.0 preview 6 build @omajid did for Fedora 30 includes the desktop targeting pack.

Reply:
> desktop: well, we don't currently plan to make this targeting pack available on Linux because there are some build-time Windows dependencies:

@dleeapho @crummel Any ideas about this? Should source-build be building `Microsoft.WindowsDesktop.App.Ref` on Linux?

Reply:
@dagood do you know why we have the windows targeting pack in our source-build preview6? Does it need a fix somewhere?

Reply:
The intent is definitely not to build it: https://github.com/dotnet/core-setup/blob/master/src/pkg/projects/dir.props#L32-L35. But I'm also wondering, where are you seeing it, what type of artifact?

This is the new nupkgs that source-build infra detects, at least--no windowsdesktop:

```
  [23:04:07.40] Building 'core-setup' to produce tarball...done
  New NuGet package(s) after building core-setup:
    -> Microsoft.DotNet.PlatformAbstractions 3.0.0-preview6-27804-01
    -> Microsoft.Extensions.DependencyModel 3.0.0-preview6-27804-01
    -> Microsoft.NET.HostModel 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.App 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.App.Host.centos.7-x64 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.App.Internal 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.App.Ref 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.DotNetAppHost 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.DotNetHost 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.DotNetHostPolicy 3.0.0-preview6-27804-01
    -> Microsoft.NETCore.DotNetHostResolver 3.0.0-preview6-27804-01
    -> NETStandard.Library.Ref 2.1.0-preview6-27804-01
    -> runtime.centos.7-x64.Microsoft.NETCore.App 3.0.0-preview6-27804-01
    -> runtime.centos.7-x64.Microsoft.NETCore.DotNetAppHost 3.0.0-preview6-27804-01
    -> runtime.centos.7-x64.Microsoft.NETCore.DotNetHost 3.0.0-preview6-27804-01
    -> runtime.centos.7-x64.Microsoft.NETCore.DotNetHostPolicy 3.0.0-preview6-27804-01
    -> runtime.centos.7-x64.Microsoft.NETCore.DotNetHostResolver 3.0.0-preview6-27804-01
```

Searching the logs, I see they're created as reference packages, but those are only meant for source-build bits to compile against:

```
  Created '/tb/tarball_246406/reference-packages/packages/Microsoft.WindowsDesktop.App.Ref.3.0.0-preview6-27804-01.nupkg'
  Created '/tb/tarball_246406/reference-packages/packages/Microsoft.WindowsDesktop.App.3.0.0-preview6-27804-01.nupkg'
```

(I'm a little surprised to see those there, though. The annotated prebuilt doesn't even say where they're used. Filed https://github.com/dotnet/source-build/issues/1128 to figure that out.)

Reply:
Our source-build 3.0 preview6 has a Microsoft.WindowsDesktop.App.Ref folder:
```
$ ls /usr/lib64/dotnet/packs
Microsoft.AspNetCore.App.Ref              Microsoft.NETCore.App.Ref         NETStandard.Library.Ref
Microsoft.NETCore.App.Host.fedora.30-x64  Microsoft.WindowsDesktop.App.Ref
```

Reply:
I see... Core-SDK is downloading the WindowsDesktop packages in the production build, which is a bug. Filed https://github.com/dotnet/core-sdk/issues/2987.

Reply:
@omajid Shipping the downloaded version is for sure a bug, I think we probably don't have to build it at all since we shouldn't be building anything Windows-specific.